tests/Feature/NotesTest.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Feature;
+
+use App\Models\User;
+use Tests\TestCase;
+
+class NotesTest extends TestCase
+{
+    /**
+     * A basic feature test example.
+     */
+    public function testExample(): void
```

Comment:
We don't need this test - it's just example test and in this file we're testing stuff related to notes.